### PR TITLE
Refs #18990 - Create variable for table row-select width

### PR DESCRIPTION
**Pull request by johnpmitsch**
We can use this variable in the css for
https://github.com/Katello/katello/pull/6698

### CODE REVIEW

**Comment by waldenraines** [comment]
[test]

File: app/assets/stylesheets/bastion/tables.scss [hunk 1]
```diff
@@ -20,8 +20,8 @@
   
   th.row-select,
   td.row-select {
-    width: 30px;
-    max-width: 30px;
+    width: $row-select-width;
```
Comment (waldenraines) [inline app/assets/stylesheets/bastion/tables.scss]: I guess `variables.scss` needs to be imported?  @ehelms do you have any ideas about why the tests are failing?

Comment (ehelms) [inline app/assets/stylesheets/bastion/tables.scss]: Yea, it needs to be imported at the top here

**Comment by waldenraines** [comment]
I know I just approved this but I think you're going to have to open a bastion issue and update your commit message to reference it to make @ehelms happy 😄

**Pull request merged**